internal/postprocessor/.gitignore

```diff
@@ -0,0 +1 @@
+postprocessor
```

Comment:
Why is this needed?

Reply:
If you run `go build` in this directory it produces a binary called postprocessor, this makes sure that never gets uploaded to github